src/main/java/org/apache/mesos/state/AbstractStateStore.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.mesos.state;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+
+/**
+ * Abstract StateStore implementation.
+ */
+public abstract class AbstractStateStore implements StateStore {
+    private static final Logger log = LoggerFactory.getLogger(SchedulerState.class);
+
+    public void storePropertyAsObj(final String key, final Object obj) throws StateStoreException {
+        ByteArrayOutputStream bos = new ByteArrayOutputStream();
+        ObjectOutputStream oos = null;
+        try {
+            oos = new ObjectOutputStream(bos);
```

Comment:
If this is just to transform boolean (isSuppressed) to byte array and back.  We could defer this complexity with something like:

```
boolean vIn = true;
byte vOut = (byte)(vIn?1:0);
```